src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css

```diff
@@ -1,4 +1,3 @@
 .inputWrapper > div > div > input {
-  font-family: var(--family-techie);
   font-weight: var(--weight-medium);
 }
```

Comment:
Why was this removed?

@alicjakujawa specifically added this at some point, and I also believe it was cleared with Karol.

![Screenshot from 2021-02-05 14-36-51](https://user-images.githubusercontent.com/1193222/107037482-cd904580-67c3-11eb-8107-6cac4217a0e0.png)

Reply:
I removed it due to this change referencing this input: `Use of wrong typography for input content, which is, misaligned`

It's in vol.2 of the changes. I think this doc in particular was written by Jack right? That would explain why I was asked to change the typo even though you guys talked with Karol about it

Reply:
But we discussed this with Karol at some point, and agreed that techie was better here. 

Please change it back

Reply:
Sure :+1: